src/shared-validator.cc

```diff
@@ -801,10 +793,6 @@ Result SharedValidator::OnDrop(const Location& loc) {
 Result SharedValidator::OnElemDrop(const Location& loc, Var segment_var) {
   Result result = Result::Ok;
   expr_loc_ = &loc;
-  // TODO: Remove, elem.drop doesn't require a table.
-#if 1
-  result |= CheckTableIndex(Var(0, loc), Opcode::ElemDrop);
```

Comment:
I wonder if this means we are missing a couple of spec tests?

Reply:
Hm, good point.

Reply:
https://github.com/WebAssembly/bulk-memory-operations/pull/140